defaults/main.yml

```diff
@@ -19,6 +19,8 @@ redis_nofile_limit: 16384
 
 ## Role options
 # Configure Redis as a service
+redis_server: true
```

Comment:
I think the variable is wrongly placed, it should be just before `redis_sentinel`, with a comment explaining what it doestransient file system.